lib/screens/splash/splash.dart

```diff
@@ -0,0 +1,63 @@
+import 'dart:async';
+import 'package:ecellapp/core/res/strings.dart';
+import 'package:ecellapp/core/utils/injection.dart';
+import 'package:ecellapp/screens/home/home.dart';
+import 'package:ecellapp/screens/login/cubit/login_cubit.dart';
+import 'package:ecellapp/screens/login/login.dart';
+import 'package:ecellapp/screens/login/login_repository.dart';
+import 'package:ecellapp/widgets/eCellAnimation.dart';
+import 'package:ecellapp/widgets/screen_background.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+import 'package:shared_preferences/shared_preferences.dart';
+
+class SplashScreen extends StatefulWidget {
+  @override
+  _SplashScreenState createState() => _SplashScreenState();
+}
+
+class _SplashScreenState extends State<SplashScreen> {
+  final splashDelay = 3;
+
+  @override
+  void initState() {
+    super.initState();
+    _loadWidget();
+  }
+
+  _loadWidget() async {
+    var _duration = Duration(seconds: splashDelay);
+    return Timer(_duration, navigationPage);
+  }
```

Comment:
1. no need to create small functions and call them. place these lines in `initState` directly.
2. don't create unwanted variables
3. call the `init()` function of main.dart (line 9) here.
```suggestion
  _loadWidget() async {
    await init();
    Timer(Duration(seconds: splashDelay), navigationPage);
  }
```

Reply:
done

Reply:
`1.` is not done yet.